corporate/views.py

```diff
@@ -66,15 +66,42 @@ def payment_method_string(stripe_customer: stripe.Customer) -> str:
     # a customer via the Stripe dashboard.
     return _("Unknown payment method. Please contact %s." % (settings.ZULIP_ADMINISTRATOR,))  # nocoverage
 
+@zulip_login_required
```

Comment:
For API endpoints, you don't want or need `@zulip_login_required`.  We might need `@require_realm_admin` if that's appropriate here.